pkg/proxy/proxy.go

```diff
@@ -289,6 +289,34 @@ func (p *Proxy) DeleteRelease(name, namespace string, purge bool) error {
 	return nil
 }
 
+// TestRelease runs tests for a release in a namespace
+func (p *Proxy) TestRelease(name, namespace string) (*TestStatus, error) {
+
+	// Validate that the release actually belongs to the namespace
+	release, err := p.GetRelease(name, namespace)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to locate release: %v", err)
+	}
+
+	// Request Tiller to run tests for the specified release
+	// error channel (second return value) is ignored for now since all relevant "errors"
+	// are channeled into "testReleaseResponseChannel"
+	testReleaseResponseChannel, _ := p.helmClient.RunReleaseTest(release.GetName(), helm.ReleaseTestCleanup(true))
```

Comment:
we should not ignore the error channel, there is where you are going to receive errors in case it cannot connect